### ISSUE TITLE
Launcher attempts to write config to system32

### ISSUE THREAD

**Issue by conkeat** [griffenx/Apex-Launcher#22]
# Exception Details
System.UnauthorizedAccessException: Access to the path 'C:\WINDOWS\system32\config.txt' is denied.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode)
   at Apex_Launcher.Program.initialize()
   at Apex_Launcher.Program.Main()

# Configuration
* Current Launcher Version: 1.4.3.0

# Files

# Environment
* Locale: en-US
* Operating System: Microsoft Windows NT 6.2.9200.0

**Comment by griffenx** [comment]
Why was this issue closed? Can you provide more details as to what happened?

**Comment by conkeat** [comment]
I closed because I couldn't play the game a message kept popping up telling me to post so I thought if I did it would go away. After I did I tried again and the message was gone, so I closed it and kept going. What happened was I was in full screen mode and I hit soft reset. Then the game crashed and gave me the message. Sorry I can't go into more detail it happened awhile ago. Would you like me to reopen this?

**Comment by griffenx** [comment]
Issues should only be closed if they have been solved. I'm a bit confused as to what you are describing. Can you walk me through the process that resulted in this error?

**Comment by conkeat** [comment]
Below are the screenshots that I hope clear this up. I set my options to full screen and then occasionally if I soft reset by pressing Fn then F12. When that happens it might work and load back up like normal but sometimes I will get that critical error. Once I click okay the player closes.
![screenshot 43](https://user-images.githubusercontent.com/23084737/34690997-3fa4bb30-f489-11e7-9dab-98245e13e293.png)

![screenshot 45](https://user-images.githubusercontent.com/23084737/34691009-499647ee-f489-11e7-9e27-c5988903525a.png)

**Comment by griffenx** [comment]
This isn't the error shown above. In-game issues should be reported on the subreddit. How did you manage to trigger the crash in the launcher?

**Comment by griffenx** [comment]
I believe this is a duplicate of #20, but since the author of that issue never provided any details, I'll consider this the primary issue report, unless you fail to provide an update.

**Comment by conkeat** [comment]
I'm not sure why the crash occurred from the launcher. I downloaded the game and set everything to install in its default locations. I started up the game and played fine in full screen up to the point of picking a starter. I saved before choosing and then soft reset. The critical error showed up from the screenshots, I clicked okay and it closed out of the game. I went back to the launcher and I don't remember if I could open it and click launch game or if the launcher wouldn't open at all. Whichever it was a different window popped up with the original error message and a prompt asking me to post to github. I did and it kept crashing afterwards. I left the game alone for awhile, without moving any of the files or changing anything. I tried again later and it just started working again without crashing. That's the full story as best I can remember. I will try playing through to that point to see if I can recreate the error.
